<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSinglePoint</class>
 <widget class="QDialog" name="DialogSinglePoint">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Single point</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../share/resources/icon.qrc">
    <normaloff>:/icon/64x64/icon64x64.png</normaloff>:/icon/64x64/icon64x64.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>241</width>
     <height>129</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Coordinates on the sheet</string>
   </property>
   <property name="title">
    <string>Coordinates</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLabel" name="labelYCor">
      <property name="text">
       <string>Y coordinate</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="labelXCor">
      <property name="text">
       <string>X coordinate</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QDoubleSpinBox" name="doubleSpinBoxX"/>
    </item>
    <item row="1" column="0">
     <widget class="QDoubleSpinBox" name="doubleSpinBoxY"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelEditName">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>159</red>
         <green>158</green>
         <blue>158</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string>Point label</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>9</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>221</width>
     <height>27</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditName</tabstop>
  <tabstop>doubleSpinBoxX</tabstop>
  <tabstop>doubleSpinBoxY</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../share/resources/icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSinglePoint</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSinglePoint</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
